sdk/python/readme.py

```diff
@@ -301,19 +301,50 @@ def write_notebook_workflow(
 
     if not ("automl" in folder):
         workflow_yaml += f"""
-          papermill -k python {name}.ipynb {name}.output.ipynb
+          papermill -k python {name}.ipynb {name}.output.ipynb > sample_log.txt 2>&1
```

Comment:
question: Would grepping the output notebook not work? It has all the execution output.

Reply:
Good point. Yeah, for notebooks we can probably discard the sample_log.txt. I'll give it a try.